truss/main.go

```diff
@@ -145,40 +221,40 @@ func mkdir(path string) error {
 	return err
 }
 
+// exitIfError will print the error message and exit 1 if the passed error is
+// non-nil
 func exitIfError(err error) {
 	if errors.Cause(err) != nil {
 		defer os.Exit(1)
 		fmt.Printf("%v\n", err)
 	}
 }
 
-// readPreviousGeneration accepts the path to the directory where the inputed .proto files are stored, protoDir,
-// it returns a []truss.NamedReadWriter for all files in the service/ dir in protoDir
-func readPreviousGeneration(protoDir, packageName string) ([]truss.NamedReadWriter, error) {
-	dir := protoDir + "/" + packageName + "-service"
-	if fileExists(dir) != true {
+// readPreviousGeneration returns a []truss.NamedReadWriter for all files serviceDir
+func readPreviousGeneration(serviceDir string) ([]truss.NamedReadWriter, error) {
+	if fileExists(serviceDir) != true {
 		return nil, nil
 	}
 
 	var files []truss.NamedReadWriter
 	sfs := simpleFileConstructor{
-		protoDir: protoDir,
-		files:    files,
+		dirPath: filepath.Dir(serviceDir),
```

Comment:
The `directory` and `path` names have always confused me.
>A **directory** is a "folder", a place where you can put files or other directories (and special files, devices, symlinks...). It is a **container for filesystem objects**. A **path** is a **string** that specify how to reach a filesystem object...

So `dirPath` is saying the path to the directory like filepath is saying the same thing for a file. 

Because of this I was really tempted to change `serviceDir` and such to `serviceDirPath` but that is so verbose, even if it is technically correct.

Just calling it `path` though seemed bad because at the time I was importing `path`. 

Do you have an alternative name suggestion?